docs/subitem-udw-in-sf-app.md

```diff
@@ -0,0 +1,178 @@
+# Reuse PlatformUI 1.x YUI based components in PlatformUI 2.x
+
+* status: draft
+* author: dp@ez.no
+
+## Idea
+
+Expose some YUI based component as WebComponents. Relying on 1.x code would then
+be a temporary solution and progressively we can get rid of YUI based code while
+still keeping the WebComponents. Other said, the custom element would become a
```

Comment:
>  Other said, the custom element would become a public API to use those components.

Maybe => "Said in other words, the custom element servers as a public api _(hiding implementation details for those consuming them)_."